### PR TITLE
[e2e] refactoring of assertions cross tests #2437

### PR DESCRIPTION
**Pull request by ArthurNiedial**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [x] No need to
- [ ] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 1]
```diff
@@ -79,16 +82,16 @@ public void updateTopic() {
...
         pages.openTopicView(CLUSTER_NAME, TOPIC_TO_UPDATE)
-                .openEditSettings()
-                // Assertions
-                .cleanupPolicyIs(COMPACT_POLICY_VALUE)
-                .timeToRetainIs(UPDATED_TIME_TO_RETAIN_VALUE)
-                .maxSizeOnDiskIs(UPDATED_MAX_SIZE_ON_DISK)
-                .maxMessageBytesIs(UPDATED_MAX_MESSAGE_BYTES);
+                .openEditSettings();
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(COMPACT_POLICY_VALUE).isEqualTo(pages.topicsList.openTopic(TOPIC_TO_UPDATE).openEditSettings().cleanupPolicyIs(COMPACT_POLICY_VALUE));
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:90]: remove please 'pages.topicsList.openTopic(TOPIC_TO_UPDATE).openEditSettings()' from assertions

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 2]
```diff
@@ -113,15 +116,16 @@ public void deleteTopic() {
     @Test
     void produceMessage() {
         pages.openTopicsList(CLUSTER_NAME)
-                .waitUntilScreenReady()
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:116]: why did you remove this step?

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 1 above)
Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:90]: @ArthurNiedial why do all soft assertions from this method use regular assertions from TopicCreateEditSettingsView.class. Does SoftAssertions library really works this way?

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java [hunk 3]
```diff
@@ -31,7 +28,7 @@ public static void beforeAll() {
         ApiHelper apiHelper = Helpers.INSTANCE.apiHelper;
 
         String connectorToDelete = getResourceAsString("delete_connector_config.json");
-        String connectorToUpdate = getResourceAsString("config_for_create_connector_via_api.json");
+        String connectorToUpdate = getResourceAsString("config_for_create_connector.json");
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java]: Fixed

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/topic/TopicCreateEditSettingsView.java [hunk 4]
```diff
@@ -158,7 +176,7 @@ public TopicCreateEditSettingsView maxMessageBytesIs(String bytes) {
     }
 
 
-    private static class KafkaUISelectElement {
+    protected static class KafkaUISelectElement {
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/topic/TopicCreateEditSettingsView.java]: Fixed

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaView.java [hunk 5]
```diff
@@ -18,16 +18,16 @@ public SchemaView waitUntilScreenReady() {
     }
 
     @Step
-    public SchemaView isCompatibility(CompatibilityLevel.CompatibilityEnum compatibility){
-        $x("//div//p[.='" + compatibility.getValue() + "']").shouldBe(Condition.visible);
-        return this;
+    public boolean isCompatibility(CompatibilityLevel.CompatibilityEnum compatibility){
+     return    $x("//div//p[.='" + compatibility.getValue() + "']").is(Condition.visible);
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaView.java:22]: Fixed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 6]
```diff
@@ -86,8 +86,8 @@ void updateSchemaAvro() {
                 .selectCompatibilityLevelFromDropdown(CompatibilityLevel.CompatibilityEnum.NONE)
                 .setNewSchemaValue(fileToString(PATH_AVRO_FOR_UPDATE))
                 .clickSubmit()
-                .waitUntilScreenReady()
-                .isCompatibility(CompatibilityLevel.CompatibilityEnum.NONE);
+                .waitUntilScreenReady();
+        Assertions.assertTrue(pages.schemaRegistry.openSchema(SCHEMA_AVRO_API_UPDATE).isCompatibility(CompatibilityLevel.CompatibilityEnum.NONE),"isCompatibility()");
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java]: Fixed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 1 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java]: As we agree i plased "TODO" to refactor assertions to settings tab using softassertions .

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaView.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaView.java:22]: Really? Can't see any changes

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 1 above)
Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java]: UPD: issue causing test were disabled resolved. need to update assertions

**Review by VladSenyuta — COMMENTED** [review]
not all comments were fixed
